integration_tests/sdk/test-config-example.yml

```diff
@@ -28,3 +31,4 @@ data:
 # `aqueduct_engine`, which is our native orchestration engine.
 compute:
   aqueduct_engine:
+    conda: test_conda
```

Comment:
And there doesn't need to be an entry for conda in `test-credentials.yml` right? Can we document that here actually.